MIPX/MIPX.md

```diff
@@ -20,103 +20,111 @@ Replaces: N/A
+
+**Platform Manager:** An individual or organization that runs the technical infrastructure that serves content to an asset held by an Asset Owner. This individual or entity may be outside MakerDAO's direct control. For example, an individual or organization hosting the webserver that serves content on a given domain is a platform manager. As another example, a centralized organization such as Twitter or Github that manages a publicly available content platform is a Platform Manager.
+
+#### Assumptions
+The following assumptions are made regarding control and ownership of Intangible Assets within MakerDAO in order for this MIP to be effectively used to manage the access of digital assets.
 
 **Assumption #1**
-MIPX assumes that some owning entity exists which retains 'owner-control' of DAO owned digital assets. 'Owner-control' is defined as the power to forcably remove control of a digital asset from a controlling entity and transfer it to a new controlling entity. 
+MIPX assumes that some Asset Owner exists for the assets managed using this MIP. 
 
-At the time of writing this MIP, the 'owning entity' for many of MakerDAO's digital assets is the DAI Foundation.
+At the time of writing this MIP, the Asset Owner for many of MakerDAO's Intangible Assets is the DAI Foundation.
 
 **Assumption #2**
-MIPX assumes that MakerDAO actors usually have positive relationships with one another and are able to come to mutually satisfactory resolutions in almost all cases of modification of control.
+MIPX assumes that actors under the umbrella of MakerDAO usually have positive relationships with one another and are able to come to mutually satisfactory resolutions in the majority of situations that require a modification to the Content Managers or Platform Managers of an Intangible Asset.
 
 **Assumption #3**
-MIPX assumes that there are sufficient MakerDAO actors to meet the transparency requirements defined in MIPXc3.
+MIPX assumes that there are sufficient Core Unit Facilitators to meet the transparency requirements defined in MIPXc3.
 
-### MIPXc2: Digital Asset List
+### MIPXc2: Intangible Asset List
 
-MIPXc2 defines a list of the digital assets owned by MakerDAO and should be managed by the MIP Editors. 
+MIPXc2 defines a list of the Intangible Assets owned by MakerDAO and should be managed by the MIP Editors. 
 
 #### vote.makerdao.com
 > **Asset Type:** Subdomain  
-> **Controller:** DUX Incubating Core Unit  
+> **Asset Owner:** DAI Foundation  
+> **Platform Manager:** DUX Incubating Core Unit (DUX email?)
+> **Content Manager:** GovAlpha Core Unit (GovAlpha@protonmail.com)
 > **Purpose:** Provide a safe and reliable way for Maker Governance to take part in the op-chain governance of MakerDAO.  
+> **Last Modification:** N/A (Grandfathered)  
 
 etc...
 
-### MIPXc3: Agreed Modification of Control
+### MIPXc3: Agreed Modification of Management
```

Comment:
This has been listed as a process in the definitions, yet it is not introduced as a one in its opening lines, while MIPxc4 is. I assume that's probably because this component does not use a subproposal but rather a forum post (to be posted where exactly?)?

I think this component would work better with a new intro that maybe makes it clear that this component is defining a process, albeit not one instantiated through subproposals.

As it is, it's a list of conditions that rather unexpectedly happens to contain a description of the process itself.

Reply:
... maybe? On a second read, it's not too bad. I'd still reformat it a bit though.

Reply:
Mmm, it is a process, but not a subproposal?